### PR TITLE
Update Pneumatics

### PR DESCRIPTION
**Pull request by aarooshg**
We wrote the code for our pneumatics for both the elevator and the manipulator. This included writing methods containing if-then loops to show the different states of the pneumatics of the manipulator and elevator and made it as simple as possible.

### CODE REVIEW

**Review by akashsarada — CHANGES_REQUESTED** [review]
Can we make the pneumatic states as enumerators rather than booleans?

**Review by akashsarada — APPROVED** [review]
Looks good!

**Review by kelmalon — CHANGES_REQUESTED** [review]
Please add Javadocs, and Remove Manipulator Solenoid. There are only pneumatic on the elevator now.

**Review by kelmalon — APPROVED** [review]
Looks good! Ready to merge.

File: src/main/java/org/troyargonauts/Constants.java [hunk 1]
```diff
@@ -1,10 +1,9 @@
 package org.troyargonauts;
 
 public final class Constants {
-
-    public interface Manipulator {
-        public static final int kForwardChannel = 0;
-        public static final int kReverseChannel = 1;
+    public interface Elevator {
+        int kElevatorForwardChannel = 0;
```
Comment (akashsarada) [inline src/main/java/org/troyargonauts/Constants.java:5]: The constants need to be in the UPPER_CASE format, not in the Hungarian format.

**Review by akashsarada — CHANGES_REQUESTED** [review]

**Review by DarkMG1 — APPROVED** [review]

**Comment by DarkMG1** [comment]
Unneeded and outdated.

**Pull request closed without merging**